community/membership.md

```diff
@@ -0,0 +1,286 @@
+- Responsible for project quality control via code reviews
+  - Focus on holistic acceptance of contribution such as dependencies with other features, backwards / forwards
+    compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests (inactivity may result in suspension until active again)
+- Mentor contributors and reviewers
+- May approve and merge code contributions for acceptance
+
+
+
+## Lead
+
+Subproject Leads are the technical authority for a subproject in the Argo
+project.  They *MUST* have demonstrated both good judgement and responsibility
+towards the health of that subproject.  Subproject leads *MUST* set technical
+direction and make or approve design decisions for their subproject - either
+directly or through delegation of these responsibilities.
+
+**Defined by:** *leads* entry in subproject `OWNERS` files
+
+### Requirements
+
+The process for becoming an subproject lead should be defined in the SIG
+charter of the SIG owning the subproject.  Unlike the roles outlined above, the
+Leads of a subproject are typically limited to a relatively small group of
+decision makers and updated as fits the needs of the subproject.
+
+The following apply to the subproject for which one would be an lead.
+
+- Deep understanding of the technical goals and direction of the subproject
+- Deep understanding of the technical domain of the subproject
+- Sustained contributions to design and direction by doing all of:
+  - Authoring and reviewing proposals
+  - Initiating, contributing and resolving discussions (emails, GitHub issues, meetings)
+  - Identifying subtle or complex issues in designs and implementation PRs
+- Directly contributed to the subproject through implementation and / or review
+
+### Responsibilities and privileges
+
+The following apply to the subproject for which one would be an lead.
+
+- Make and approve technical design decisions for the subproject.
+- Set technical direction and priorities for the subproject.
+- Define milestones and releases.
+  - Decides on when PRs are merged to control the release scope.
+- Mentor and guide approvers, reviewers, and contributors to the subproject.
+- Escalate reviewer and maintainer concerns (i.e. responsiveness, availability, and general contributor community health) to the TC.
+- Ensure continued health of subproject
+  - Adequate test coverage to confidently release
+  - Tests are passing reliably (i.e. not flaky) and are fixed when they fail
+- Ensure a healthy process for discussion and decision making is in place.
+- Work with other subproject leads to maintain the project's overall health and success holistically
+
+
+
+## Inactive members
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
```

Comment:
Perhaps remove "forever" here since you would never know and it's not uncommon for people to come back again when their roles and priorities change :-)

Reply:
I believe you are stating the same thing as written? 

> **are not expected to be** actively contributing forever

Reply:
Okay that's fine then.